docs/metrics.md

```diff
@@ -0,0 +1,466 @@
+# Metrics
+
+Since EventStoreDB version `v23.6.0`, some metrics in [prometheus format](https://github.com/prometheus/docs/blob/main/content/docs/instrumenting/exposition_formats.md#text-based-format) are available on the `/metrics` endpoint. Promethus can be configured to scrape this endpoint directly. The metrics are configured in `telemetryconfig.json`. In the near future we hope to publish a sample grafana dashboard.
+
+::: note
+`/metrics` does not yet contain metrics for Projections and Persistent Subscriptions. To view these in Prometheus it is still recommended to use the [Prometheus exporter](./diagnostics.md#prometheus)
+:::
+
+## Metrics Reference
+
+### Caches
+
+#### Cache hits and misses
+EventStoreDB tracks cache hits/misses metrics for `stream-info` and `chunk` caches.
+
+| Time Series | Type       | Description |
+| :---------- | :--------- | :---------- |
+| `eventstore_cache_hits_misses{cache=<CACHE_NAME>,kind=<"hits"\|"misses">}` | [Counter](#common-types) | Total hits/misses on _CACHE_NAME_ cache   |
```

Comment:
```suggestion
| `eventstore_cache_hits_misses{cache=<CACHE_NAME>,kind=<"hits"\|"misses">}` | [Counter](https://prometheus.io/docs/tutorials/understanding_metric_types/) | Total hits/misses on _CACHE_NAME_ cache   |
```
Reasoning: either link directly to the source doc in prometheus, or copy some information in that section .
the text when following that link is " go to prometheus doc" without a link to it .